pkg/cloudprovider/providers/azure/azure_standard.go

```diff
@@ -685,3 +688,13 @@ func (as *availabilitySet) EnsureBackendPoolDeleted(poolID, vmSetName string) er
 	// Do nothing for availability set.
 	return nil
 }
+
+// get a storage account by UUID
+func getStorageAccountName(accountNamePrefix string) string {
+	uniqueID := strings.ToLower(strings.Replace(string(uuid.NewUUID()), "-", "", -1))
```

Comment:
no need to call strings.ToLower here? Since will call in next line.

Reply:
fixed